style_guide.md

```diff
@@ -0,0 +1,18 @@
+# Style Guide
+
+Please follow the guidelines stated here when contributing to this repository.
+
+## Spacing
+
+- Always use tabs. Spaces suck.
+- Avoid trailing spaces at the end of lines.
+- Avoid trailing empty lines at the end of files.
+- Surround operators with a single space (except for when using `=` in parameter lists).
+- Leave two empty lines between functions.
+- Optionally leave one empty line between logical blocks of code in the same function.
```

Comment:
If it's optional why mention it? It should either be enforced or not mentioned (I think enforced)

Reply:
Logical blocks is kind of a blurry definition. This is kind of a suggestion for the devs to be aware that it should be done. Regardless, I'll remove "optionally".